code/release-2/00-metadata/emp500_s3_make_mapping_files_prep_info.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## emp500_s3_make_mapping_files_prep_info.ipynb\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This notebook takes a single sample information (metadata) file, a general prep information file, and amplicon target-specific prep information files. It generates a merged prep information file and Qiime mapping file for each amplicon target. A single Qiita study ID will be prepended to sample names when the mega-study goes into Qiita."

```

Comment:
I remain confused...

Reply:
Thanks for all of these comments, Daniel! However, as I did not make any changes to anything in 'code/' for this pull request, I would prefer that you limit requests for changes here to the relevant methods, as we need to get this up on biocore for the paper. I would prefer to consider any changes you would like me to make to anything else that is already merged with biocore at a later time, as I did not create any of that content and will need to take some time to update it. Thanks again